tests/kernel/install_klp_product.pm

```diff
@@ -19,6 +19,20 @@ use testapi;
 use utils;
 use klp;
 use power_action_utils 'power_action';
+use Utils::Architectures qw(is_s390x);
+
+sub do_reboot {
+    my $self = shift;
+
+    power_action('reboot', textmode => 1);
+    $self->wait_boot;
+
+    if (is_s390x) {
+        select_console('root-console');
+    } else {
```

Comment:
Maybe here just for sure add `prepare_serial_console()` to fix ppc64le issues on older releases?